apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/SwapWebView.tsx

```diff
@@ -12,25 +12,35 @@ import { WebviewAPI, WebviewState } from "~/renderer/components/Web3AppWebview/t
 import { initialWebviewState } from "~/renderer/components/Web3AppWebview/helpers";
 import { handlers as loggerHandlers } from "@ledgerhq/live-common/wallet-api/CustomLogger/server";
 import { TopBar } from "~/renderer/components/WebPlatformPlayer/TopBar";
+import { updateAccountWithUpdater } from "~/renderer/actions/accounts";
+import { SwapOperation } from "@ledgerhq/types-live/lib/swap";
+import BigNumber from "bignumber.js";
+import { SubAccount } from "@ledgerhq/types-live";
+import { getAccountCurrency } from "@ledgerhq/live-common/account/helpers";
+import { getAccountIdFromWalletAccountId } from "@ledgerhq/live-common/wallet-api/converters";
 
 type CustomHandlersParams<Params> = {
   params: Params;
 };
 
+export type SwapProps = {
+  provider: string;
+  fromAccountId: string;
+  fromParentAccountId: string;
```

Comment:
As it can be undefined, I would add it as optional...but up to you 
```suggestion
  fromParentAccountId?: string;
```